packages/nvidia_nat_core/src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -1263,147 +1263,165 @@ class MCPToolInfo(BaseModel):
-                        mcp_clients_info.append({
-                            "function_group": group_name,
-                            "server": "unknown",
-                            "transport": config.server.transport if config.server else "unknown",
-                            "session_healthy": False,
-                            "protected": False,
-                            "error": str(e),
-                            "tools": [],
-                            "total_tools": 0,
-                            "workflow_tools": 0
-                        })
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)